http/.env

```diff
@@ -0,0 +1 @@
+PORT=8080
```

Comment:
what is the point of having .env file and not using variables from it?

other than that, you shouldn't commit the .env file with actual data. the only file that should be committed is the .env.example file containing the mock data.

Reply:
done